sdcm/cluster.py

```diff
@@ -4491,7 +4491,7 @@ def get_test_keyspaces(self, db_node=None):
         """Function returning a list of non-system keyspaces (created by test)"""
         db_node = db_node or self.nodes[0]
         keyspaces = db_node.run_cqlsh("describe keyspaces").stdout.split()
-        return [ks for ks in keyspaces if not is_system_keyspace(ks)]
+        return [ks.strip('"') for ks in keyspaces if not is_system_keyspace(ks)]
```

Comment:
you still need to make sure all the usages are using `cql_quote_if_needed` as needed when using it in CQL statements

maybe it's time to have  `cql_unquote_if_needed` ? so it would be a bit more clear ?